src/www/cordova.webpack.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const webpack = require('webpack');
+
+const OUTPUT_BASE = path.resolve(__dirname, '../../www');
+const GENERATE_CSS_RTL_LOADER = path.resolve(__dirname, '../../scripts/rtl_css_webpack.js');
+const BABEL_LOADER = {
+  loader: 'babel-loader',
+  options: {
+    presets: [
+      '@babel/preset-env',
+    ],
+  },
+};
+
+module.exports = {
+  entry: [
+    require.resolve('@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js'),
+    require.resolve('@babel/polyfill/dist/polyfill.min.js'),
+    require.resolve('web-animations-js/web-animations-next-lite.min.js'),
+    require.resolve('@webcomponents/webcomponentsjs/webcomponents-loader.js'),
+    path.resolve(__dirname, './style.css'),
+    path.resolve(__dirname, './app/cordova_main.ts'),
+  ],
+  mode: 'production',
```

Comment:
Can we refactor the common electron/cordova build into a `base.webpack.js` file, like we do in the manager?

Reply:
Let me take another stab at this. I tried doing this, but there were enough difference that made it difficult to do cleanly.

Reply:
Done. Let me know what you think.